cdap-data-fabric/src/test/java/io/cdap/cdap/spi/data/nosql/NoSqlStructuredTableTest.java

```diff
@@ -168,6 +158,44 @@ private void testLimitIterator(int max, int limit) throws Exception {
     Assert.assertEquals(expected.subList(0, Math.min(max, limit)), actual);
   }
 
+  @Test
```

Comment:
Did we add the sort by index tests brought up by @albertshau ? Like the edge case with limititerator+sort

Reply:
https://github.com/cdapio/cdap/pull/14554#discussion_r982578777

Reply:
discussed offline - this should be handled here https://github.com/cdapio/cdap/blob/develop/cdap-storage-spi/src/test/java/io/cdap/cdap/spi/data/StructuredTableTest.java#L664